### DISCUSSION TITLE
Scratch Addons - Bookmarklet For Mobile

### DISCUSSION THREAD

**Discussion by heatnovea** [ScratchAddons/ScratchAddons#7977] [Discussion]
I know alot of us use PC but I use an ipad at home too, and I think that it would be cool for Scratch Addons to have a javascript bookmarklet for safari that opens the GUI. I think this because I love the transparency feature in scratch addons on PC but I cant always access my PC.

**Comment by WorldLanguages** [comment]
Related: "SA userscript"

**Reply by heatnovea** [reply]
I am confused lol (I know little about anything but basic js and complex html+css)
